src/coreclr/nativeaot/Runtime/wasm/pal_compiler.h

```diff
@@ -0,0 +1,74 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma once
+
+#ifndef __cplusplus
+#include <stdbool.h>
+#endif
```

Comment:
I'd like to understand the purpose of these `pal_` files. How much of them are needed? Are they supposed to be exact copies of some other files in the tree? If not - stuff like `#if defined(TARGET_ANDROID)` code could presumably be pruned.

Reply:
Think they were part of the journey that I didn't clear up.  Have removed them now (they were copies from the Globalization folder)